.nsprc

```diff
@@ -1,7 +1,2 @@
 {
-  "1111529": {
-    "active": true,
-    "notes": "Dev only dependency and only affects windows users",
-    "expiry": 1767940105000
```

Comment:
new cypress version pulls in the latests version with the fix in so no longer need to remove